partials/header.html

```diff
@@ -5,9 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>boringproxy.io</title>
     <link rel="stylesheet" type="text/css" href="/styles.css">
-    <link rel="icon" href="/logo.png">
+    <link rel="icon" href="/logo.svg">
```

Comment:
I would love to switch to using the SVG (especially for the logo baked into the boringproxy executable), but AFAIK safari still doesn't support SVG favicons.

Reply:
Okay, I will download the current PNG from the website and add it to the repository. I was simply falling back to the one file available to me.

Reply:
If you wouldn't mind using this logo (recently added to boringproxy repo) that would be ideal:

https://github.com/boringproxy/boringproxy/blob/master/default_logo.png

Reply:
You can rename it to logo.png for boringproxy.io